### PR TITLE
Backup fixes

### PR DESCRIPTION
**Pull request by rkratky**
### What does this PR do?

Fixes mistake in "Backup and Disaster Recovery".

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc [hunk 1]
```diff
@@ -9,24 +9,29 @@ summary:
 ---
 
 :page-liquid:
-:parent-context-of-persistent-volumes-backups: {context}
 
 [id="persistent-volumes-backups_{context}"]
 = Persistent Volumes backups
 
-Persistent Volumes (PVs) store the {prod-short} workspace data like a source code of a repository. 
+Persistent Volumes (PVs) store the {prod-short} workspace data in the form of a source code in a repository.
```
Comment (yhontyk) [inline src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc]: Does it really do that?
I though it drew a comparison in this sentence,  as in `it stores the data like a repository stores the code`

Comment (ibuziuk) [reply src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc]: @rkratky the statement in the fix misleading

Comment (rkratky) [reply src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc]: Thanks both. Fixing.

Comment (rkratky) [reply src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc]: @ibuziuk, would this be correct?

_...workspace data similarly to how source code is stored in a repository._

Comment (ibuziuk) [reply src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc]: well, I'm not sure I get what is the problem with the original statement - PV stores workspace data - source code, scripts, configuration files etc.

**Review by yhontyk — APPROVED** [review]

**Review by ibuziuk — APPROVED** [review]

**Pull request merged**